bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -287,7 +287,7 @@ int runCmd(CommandLine cmdLine) throws Exception {
 
         public RecoverCmd() {
             super(CMD_RECOVER);
-            opts.addOption("d", "deleteCookie", false, "Delete cookie node for the bookie.");
+            opts.addOption("d", "delete_cookie", false, "Delete cookie node for the bookie.");
```

Comment:
this would break backward compatibility?

Reply:
Thanks. will change them back.